src/events/reactions/rawReactionAddNotify.ts

```diff
@@ -96,17 +96,17 @@ export default class extends Event {
 	}
 
 	private async fetchCount(data: LLRCData, emoji: string, id: string) {
-		const users = (await api(this.client).channels(data.channel.id).messages(data.messageID).reactions(emoji).get()) as APIUser[];
+		const users = (await api(this.context.client).channels(data.channel.id).messages(data.messageID).reactions(emoji).get()) as APIUser[];
```

Comment:
Can't we make this `api().channels` and add a DI based default client where we have the api stuff?